src/main/content/_assets/js/instance-common.js

```diff
@@ -496,4 +507,47 @@ function createSVG(id, classNames, width, height, fill){
     return `<svg class="${classNames}" width="${width}" height="${height}" fill="${fill}">
         <use xlink:href="/img/carbon-icons/carbon-icons.svg#icon--${id}"></use>
     </svg>`;
+}
+
+function setEditInstanceModalData(instanceName){
+    $("#modal-edit-instance-heading").text(`Edit ${instanceName} instance`);
+    fetchAnInstance(instanceName)
+        .then(fillInstanceModalData);
+}
+
+function fillInstanceModalData(instanceJSON){
+    if (typeof instanceJSON === "undefined") {
+        console.log("instance data is undefined, cannot load instance");
+        return;
+    }
+
+    let instanceName = instanceJSON.metadata.name;
+    let codeReadyWorkspaces = instanceJSON.spec.codeReadyWorkspaces.enable ? "enabled" : "disabled";
+    let codeReadyOption = codeReadyWorkspaces === "enabled" ? "disable" : "enable";
```

Comment:
this line seems like an extra unneeded line, why the 2 lines for CodeReady? Also if `codeReadyWorkspaces` === "enabled" wouldn't we want the option to be "enable" and not "disable" ?